### PR TITLE
Update 2023.7 release blog with hyperion breaking change

### PR DESCRIPTION
**Pull request by emontnemery**
## Proposed change

Update 2023.7 release blog with hyperion breaking change from PR https://github.com/home-assistant/core/pull/80478

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_posts/2023-07-05-release-20237.markdown [hunk 1]
```diff
@@ -644,6 +644,19 @@ the backup was initiated (`%Y-%m-%d %H:%M:%S`, for example,
 
 {% enddetails %}
 
+{% details "Hyperion" %}
+
+The behavior of hyperion lights has changed:
```
Comment (c0ffeeca7) [inline source/_posts/2023-07-05-release-20237.markdown]:
```suggestion
The behavior of Hyperion lights has changed:
```

File: source/_posts/2023-07-05-release-20237.markdown [hunk 2]
```diff
@@ -644,6 +644,19 @@ the backup was initiated (`%Y-%m-%d %H:%M:%S`, for example,
...
+
+Turning off the light entity will only clear the priority you have configured during integration setup. Other HA-independent light sources in Hyperion might still be active and cause light to be emitted. In order to turn the light output off entirely regardless of active light sources, you can enable the `switch.[instance]_led_device` entity that acts as a global switch. Subsequently, external effects that are unrelated to the Hyperion light source (such as USB capture) have been removed from the light entity. To control them, enable the respective disabled-by-default switch entity.
```
Comment (c0ffeeca7) [inline source/_posts/2023-07-05-release-20237.markdown]:
```suggestion
Turning off the light entity will only clear the priority you have configured during the integration setup. Other Home Assistant independent light sources in Hyperion might still be active and cause light to be emitted. If you want to turn the light off entirely, regardless of active light sources, you can enable the `switch.[instance]_led_device` entity that acts as a global switch. 

External effects that are unrelated to the Hyperion light source (such as USB capture) have been removed from the light entity. To control them, enable the respective disabled-by-default switch entity.
```

Comment (c0ffeeca7) [reply source/_posts/2023-07-05-release-20237.markdown]: do you think we could delete the phrase _regardless of active light sources_? I find it a bit confusing.

Comment (frenck) [inline source/_posts/2023-07-05-release-20237.markdown]:
```suggestion
Turning off the light entity will only clear the priority you have configured during the integration setup. Other HA-independent light sources in Hyperion might still be active and cause light to be emitted. To turn the light output off entirely regardless of active light sources, you can enable the `switch.[instance]_led_device` entity that acts as a global switch. Subsequently, external effects unrelated to the Hyperion light source (such as USB capture) have been removed from the light entity. To control them, enable the respective disabled-by-default switch entity.
```

**Review by c0ffeeca7 — CHANGES_REQUESTED** [review]
Tiny tweaks.
Oh, I was too late.

File: source/_posts/2023-07-05-release-20237.markdown (hunk 2 above)
Comment (emontnemery) [reply source/_posts/2023-07-05-release-20237.markdown]: @sab44 Can you review the suggestion from @c0ffeeca7, I cant' judge if it's OK to remove?

Comment (Sab44) [reply source/_posts/2023-07-05-release-20237.markdown]: @emontnemery @c0ffeeca7 I think we need this sentence the way it is to emphasize that the switch will ignore any active light sources. I agree that the wording could be improved but I'm unable to come up with better phrasing right now.

**Review by c0ffeeca7 — APPROVED** [review]
LGTM :+1:

File: source/_posts/2023-07-05-release-20237.markdown (hunk 2 above)
Comment (c0ffeeca7) [reply source/_posts/2023-07-05-release-20237.markdown]: Thank you for clarifying.

**Review by frenck — APPROVED** [review]
Also LGTM, moving forward; not to keep this breaking change out of the release notes for too long.

../Frenck

**Comment by starkillerOG** [comment]
This one got me, thanks for adding it to the breaking changes already!

**Pull request merged**